README.md

```diff
@@ -33,11 +33,27 @@ Then in an initializer or before use:
 require 'gruf'
 ```
 
-Make sure to review [UPGRADING.md](https://github.com/bigcommerce/gruf/blob/master/UPGRADING.md) 
+Make sure to review [UPGRADING.md](https://github.com/bigcommerce/gruf/blob/master/UPGRADING.md)
 if you are upgrading gruf between minor or major versions.
 
 ### Client
 
+Add an initializer:
+
+```ruby
+require 'gruf'
+
+Gruf.configure do |c|
+  c.default_client_host = 'grpc.service.com:9003'
+end
+```
+
+If you don't config `default_client_host`, you need do like this:
```

Comment:
Can you say: `If you don't explicitly set `default_client_host`, you will need to pass it into the client options, like so:`